### PR TITLE
Zib-FLACCpainScale

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -51,6 +51,14 @@ This document contains release notes per zib, indicating differences with their
 * The extension EpisodOfCare-Title has been replaced by ext-EpisodeOfCare.EpisodeOfCareName because this zib concept is functionally equivalent.
 * The extensions EpisodeOfCare-DateFirstEncounter and EpisodeOfCare-DateLastEncounter from the previous profile are not inherited because no functional backing exists.
 
+# zib-FLACCpainScale
+* The system and code value has changed to `http://snomed.info/sct` and `108291000146105` on `Observation.code`.
```
Comment (ArdonToonstra) [inline release-notes.md:55]: Would be nice to know the original code. Moreover, the system nd code do not have to be in code syntax " ' ". The code value can be italic.

File: release-notes.md [hunk 2]
```diff
@@ -51,6 +51,14 @@ This document contains release notes per zib, indicating differences with their
...
+* The system and code value has changed to `http://snomed.info/sct` and `12013003` on `Observation.component:face.code`.
+* The system and code value has changed to `http://snomed.info/sct` and `12013004` on `Observation.component:legs.code`.
```
Comment (ArdonToonstra) [inline release-notes.md:57]: I can't find the 12013003 code on snomed: https://terminologie.nictiz.nl/art-decor/snomed-ct?conceptId=12013004

It's deprecated. Is this a known issue at the zib centre?

File: resources/zib/zib-FLACCpainScale.xml [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:face.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="Face" />
+      <definition value="Observation Faces;&#xD;&#xA;The extent to which the child’s expression on the face is tensed, grimace, tension of jaws." />
+      <alias value="Gezicht" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
```
Comment (ArdonToonstra) [inline resources/zib/zib-FLACCpainScale.xml:149]: Missing valueset binding.

**Review by ArdonToonstra — CHANGES_REQUESTED** [review]

File: release-notes.md (hunk 2 above)
Comment (Ashwin-nictiz) [reply release-notes.md:57]: Created a ticket https://bits.nictiz.nl/browse/ZIB-1628

Comment (Ashwin-nictiz) [reply release-notes.md:57]: The codes are from here: https://zibs.nl/wiki/Zib_Codestelsels#Codesysteem_ScoreObservaties changed the system value

File: release-notes.md (hunk 1 above)
Comment (Ashwin-nictiz) [reply release-notes.md]: Fixed

File: release-notes.md [hunk 4]
```diff
@@ -51,6 +51,14 @@ This document contains release notes per zib, indicating differences with their
 * The extension EpisodOfCare-Title has been replaced by ext-EpisodeOfCare.EpisodeOfCareName because this zib concept is functionally equivalent.
 * The extensions EpisodeOfCare-DateFirstEncounter and EpisodeOfCare-DateLastEncounter from the previous profile are not inherited because no functional backing exists.
 
+# zib-FLACCpainScale
+* The system value on the `Observation.component.code` slices has changed from http://loinc.org to urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1.
```
Comment (ArdonToonstra) [inline release-notes.md:55]: Could you explain what this oid stands for?

File: release-notes.md [hunk 5]
```diff
@@ -51,6 +51,14 @@ This document contains release notes per zib, indicating differences with their
...
+* The code on `Observation.component:face.code` has changed from *38216-8* to *12013003*.
+* The code on `Observation.component:legs.code` has changed from *38217-6* to *12013004*.
+* The code on `Observation.component:activity.code` has changed from *38218-4* to *12013005*.
+* The code on `Observation.component:cry.code` has changed from *38219-2* to *12013006*.
+* The code on `Observation.component:consolability.code` has changed from *38220-0* to *12013009*.
+
```
Comment (ArdonToonstra) [inline release-notes.md:61]: The code on `Observation.code` has changed too?

File: known-issues.yml [hunk 6]
```diff
@@ -144,6 +144,15 @@ zib-DrugUse:
    Observation.component:routeOfAdministration.value[x]:
       - cardinality: 0..1
         reason: The cardinality mismatch between the zib (0..*) and FHIR (0..1) is explained by the parent .component element that may exists 0..* times, thereby this mapping is still honoring the cardinality requirements of the zib.
+        
+zib-FLACCpainScale:
+  zib deviations:
+   Observation:
+      - short: FLACCpainScale
+        reason: The root concept name 'FLACCpainScore' is an mistake in the zib and will be fixed at the next pre-publication.
+   Observation.note.text:
```
Comment (ArdonToonstra) [inline known-issues.yml:153]: Can you merge main into this branch and update the known-issues.yml accordingly?

File: release-notes.md (hunk 5 above)
Comment (Ashwin-nictiz) [reply release-notes.md:63]: Fix!

File: known-issues.yml (hunk 6 above)
Comment (Ashwin-nictiz) [reply known-issues.yml]: Fix!

File: release-notes.md (hunk 4 above)
Comment (Ashwin-nictiz) [reply release-notes.md]: Release notes improvement

**Review by LuudSlagter — APPROVED** [review]

File: examples/nl-core-FLACCpainScale-01.xml [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+      <code>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1"/>
+            <code value="12013003"/>
+            <display value="FLACCpijnScore Gezicht"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.21.1"/>
```
Comment (Ashwin-nictiz) [inline examples/nl-core-FLACCpainScale-01.xml:35]: The urn oid code is 2.16.840.1.113883.2.4.3.11.60.40.2.12.13.1 in the zib? Where does this value come from?

**Pull request merged**